### ISSUE TITLE
Adding new addons should work automatically with other instances

### ISSUE THREAD

**Issue by Floppy** [theodi/bothan#163]
If my instance is auto-deploying from github, adding a new addon (i.e. pusher) should automatically happen. Not sure if Heroku supports this, but perhaps there is a configuration thing we can fix.

**Comment by pezholio** [comment]
Yeah, I don't think it does - see http://stackoverflow.com/a/38753156/452684

That said, could we add this as a post deploy task? Like running `heroku addons:create pusher:free`, but whether this would work on the actual Heroku instance itself I don't know

**Comment by pezholio** [comment]
Or maybe we've reached the limit of what we can do using app.json, and we should switch to using the [Heroku Platform API](https://devcenter.heroku.com/articles/setting-up-apps-using-the-heroku-platform-api)?

**Comment by Floppy** [comment]
@pezholio bothan-deploy does this now, right?

**Comment by pezholio** [comment]
Not yet, but it's unlikely Bothan is going to evolve any more to need any new addons anyway

**Comment by quadrophobiac** [comment]
@pezholio could we close this ticket per your observation that current Bothan is not going to need any new addons - and withhold the question of whether its possible or not to some future subsequent iteration of bothan that necessitates its consideration?

**Comment by Floppy** [comment]
It's low priority for sure.

**Comment by quadrophobiac** [comment]
PR https://github.com/theodi/bothan/pull/280 is wrangling with the same issues
